cmake/Modules/FindQhull.cmake

```diff
@@ -61,34 +109,35 @@ endif()
 
 get_filename_component(QHULL_LIBRARY_DEBUG_NAME "${QHULL_LIBRARY_DEBUG}" NAME)
 
-if(QHULL_INCLUDE_DIR AND QHULL_LIBRARY)
-
-  # Include directories
-  set(QHULL_INCLUDE_DIRS ${QHULL_INCLUDE_DIR})
-  unset(QHULL_INCLUDE_DIR)
-  mark_as_advanced(QHULL_INCLUDE_DIRS)
-
-  # Libraries
-  set(QHULL_LIBRARIES optimized ${QHULL_LIBRARY} debug ${QHULL_LIBRARY_DEBUG})
-  unset(QHULL_LIBRARY)
-  unset(QHULL_LIBRARY_DEBUG)
-  mark_as_advanced(QHULL_LIBRARIES)
-
-endif()
-
 include(FindPackageHandleStandardArgs)
 find_package_handle_standard_args(Qhull
   FOUND_VAR QHULL_FOUND
-  REQUIRED_VARS QHULL_LIBRARIES QHULL_INCLUDE_DIRS
+  REQUIRED_VARS QHULL_LIBRARY QHULL_INCLUDE_DIR
 )
 
 if(QHULL_FOUND)
   set(HAVE_QHULL ON)
```

Comment:
We are missing this in the "modern config" branch.

Reply:
Good catch, I will fix it.

Reply:
Should be fixed